Baby Monitor/Source Files/Services/MediaPlayer/NodeCapture.swift

```diff
@@ -22,14 +17,21 @@ final class AudioKitNodeCapture: NSObject {
     private var node: AKNode?
     private let bufferSize: UInt32
     private var internalAudioBuffer: AVAudioPCMBuffer
-    private let bufferFormat: AVAudioFormat
+    private let inputBufferFormat: AVAudioFormat
     private let bufferQueue = DispatchQueue(label: "co.netguru.netguru.babymonitor.AudioKitNodeCapture.bufferQueue", qos: .default)
+    private let machineLearningFormat: AVAudioFormat
+    private let formatConverter: AVAudioConverter
     
-    init(node: AKNode? = AudioKit.output, bufferSize: UInt32 = 264600) throws {
+    init(node: AKNode? = AudioKit.output, bufferSize: UInt32 = MachineLearningAudioConstants.bufferSize) throws {
         self.node = node
         self.bufferSize = bufferSize
-        self.bufferFormat = node?.avAudioUnitOrNode.inputFormat(forBus: 0) ?? AVAudioFormat(commonFormat: .pcmFormatFloat32, sampleRate: 44100.0, channels: 1, interleaved: false)!
-        self.internalAudioBuffer = AVAudioPCMBuffer(pcmFormat: bufferFormat, frameCapacity: bufferSize)!
+        self.inputBufferFormat = node?.avAudioUnitOrNode.inputFormat(forBus: 0) ?? AVAudioFormat(commonFormat: .pcmFormatFloat32, sampleRate: 48000.0, channels: 1, interleaved: false)!
```

Comment:
`sampleRate` is different than the one declared in `MachineLearningAudioConstants` - wdyt about lifting it out to another constant to avoid confusion?

Reply:
Fixed it by just removing 😄

Reply:
:glowkapracuje: